### PR TITLE
Update master to reflect changes in develop

### PR DESCRIPTION
**Pull request by mexanick**
## Description

I believe we are ready to merge the latest changes from develop to master and issue a release based on master branch before we move to packaging tools. 
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context

Master is long behind the develop and should be updated. The stable releases should be based on master.
## How Has This Been Tested?

The develop branch has been extensively tested on various setups
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jsturdy — COMMENTED** [review]
Can you resolve the conflicts (use the web GUI if they're simple enough)

File: python/daq_suite/daq_suite/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['kvm-s3562-1-ip151-107.cms','cosmicstandtif.cern.ch','gem904daq01.cern.ch']
+ALLOWED_HOSTS = ['kvm-s3562-1-ip151-107.cms','gem904qc8daq.cern.ch','gem904daq01.cern.ch', 'gem904qc8daq']
```
Comment (jsturdy) [inline python/daq_suite/daq_suite/settings.py:28]: if tagging for production, I'd try to get the regex working here

File: python/daq_suite/daq_suite/urls.py [hunk 2]
```diff
@@ -29,4 +29,4 @@
     url(r'^monitoring/(\w.+)/', read_gem_system_module),
 ]
 parseXML()
-rpc_connect("eagle34")
+rpc_connect("eagle26")
```
Comment (jsturdy) [inline python/daq_suite/daq_suite/urls.py:32]: if tagging for production, i'd remove hardcoded birdnames

File: python/reg_interface/gbt.py [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    ENDC    = '\033[0m'  
+
+ADDR_IC_ADDR = None
+ADDR_IC_WRITE_DATA = None
+ADDR_IC_EXEC_WRITE = None
+ADDR_IC_EXEC_READ = None
+
+ADDR_LINK_RESET = None
+
+ADDRESS_TABLE_SLOW_CTRL_ONLY = '/mnt/persistent/texas/gem_amc_top_SLOW_CTRL_ONLY.xml'
```
Comment (jsturdy) [inline python/reg_interface/gbt.py]: system setup location?

**Comment by mexanick** [comment]
I think it is done now

**Review by jsturdy — CHANGES_REQUESTED** [review]
Aren't several of the files in this migrated to `reg_utils` now? or is this a next step?
E.g., `gbt.py`, `sca.py`, `ri_prompt.py`, `rw_reg.py`, `reg_interface.py`

File: python/daq_suite/daq_suite/settings.py (hunk 1 above)
Comment (mexanick) [reply python/daq_suite/daq_suite/settings.py:28]: this is not yet finished... I want to do a final merge before we move to reg_utils separation

File: python/daq_suite/daq_suite/urls.py (hunk 2 above)
Comment (mexanick) [reply python/daq_suite/daq_suite/urls.py:32]: same as above - I didn't invent yet a good way to handle this

File: python/reg_interface/gbt.py (hunk 3 above)
Comment (mexanick) [reply python/reg_interface/gbt.py]: It is actually not used, so removed

**Review by jsturdy — APPROVED** [review]

**Pull request merged**